rfc6962-bis.xml

```diff
@@ -1654,13 +1672,12 @@ this can be done.
 Merkle inclusion proof (<xref target="get-proof-by-hash"/>) for each observed SCT.
 These checks can be asynchronous and
 need only be done once per each certificate. In order to protect the clients'
-privacy, these checks need not reveal the exact certificate to the log. Clients
-can instead request the proof from a trusted auditor (since anyone can compute
-the proofs from the log) or request Merkle inclusion proofs for a batch of
-certificates around the SCT timestamp.
+privacy, these checks need not reveal the exact certificate to the log. Instead, clients
+can request the proof from a trusted auditor (since anyone can compute
+the proofs from the log) or communicate with the log via proxies.
         </t>
         <t>
-          Violation of the append-only property can be detected by clients comparing their instances of the Signed Tree Heads. As soon as two conflicting Signed Tree Heads for the same log are detected, this is cryptographic proof of that log's misbehavior. There are various ways this could be done, for example via gossip (see <xref target="I-D.ietf-trans-gossip"/>) or peer-to-peer communications or by sending STHs to monitors (who could then directly check against their own copy of the relevant log).
+          Violation of the append-only property or the STH issuance rate limit can be detected by clients comparing their instances of the Signed Tree Heads. There are various ways this could be done, for example via gossip (see <xref target="I-D.ietf-trans-gossip"/>) or peer-to-peer communications or by sending STHs to monitors (who could then directly check against their own copy of the relevant log). A proof of misbehavior in such cases would be a series of STHs that were issued too closely together, proving violation of the STH issuance rate limit, or an STH with a root hash that does not match the one calculated from a copy of the log, proving entries in the logs were changed since that STH was issued.
```

Comment:
It doesn't prove that the entries were changed _since_ the STH was issued (i.e. it could've been before it was issued, but after the copy was taken). So, I would just say "proving violation of the append-only property".